README.md

```diff
@@ -3,31 +3,32 @@
 Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).
 
 The Wagtail guide will ultimately include:
-- Tutorials
-- How-to articles
-- Reference materials
-- Walkthroughs and visual learning materials
+
+-   Tutorials
+-   How-to articles
+-   Reference materials
+-   Walkthroughs and visual learning materials
 
 You can learn more about the documentation system [here](https://documentation.divio.com/).
 
 # Table of Contents
 
-- [Installation](#installation)
-- [Gitpod](#gitpod)
-- [Contributing](#contributing)
-- [Other Notes](#other-notes)
+-   [Installation](#installation)
+-   [Gitpod](#gitpod)
+-   [Contributing](#contributing)
+-   [Other Notes](#other-notes)
 
 # Installation
 
 We assume that you have basic knowledge of Node/Yarn/Webpack and Python/Django/Wagtail in these instructions. We recommend you develop Wagtail Guide locally on your machine using venv.
 
 #### Dependencies
 
-- Python >= 3.9
-- Poetry
-- Git
-- Node 16.*
-- [Yarn](https://yarnpkg.com/)
+-   Python >= 3.9
+-   Poetry
+-   Git
+-   Node 16.\*
```

Comment:
Oh interesting

Reply:
yeah true, I did not even look at that - not sure why Prettier changed this.